<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Verdulería</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_productos_vender">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Vender Productos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_consultar_precio">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar Precio Unitario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_compra_productos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar Compra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_consulta_inventario">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar Inventario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_datos_maestros">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Datos Maestros</string>
    </property>
   </widget>
   <!-- Nuevo botón para mostrar la lista de compras y ventas -->
   <widget class="QPushButton" name="bt_lista_compras_ventas">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista Compras y Ventas</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
